### ISSUE TITLE
Installation script. Is update required?

### ISSUE THREAD

**Issue by xanfus** [code-saturne/code_saturne#59]
```
./install_saturne.py

The 'install_saturne.py' installer script should not be run from inside the
'/home/<user>/code_saturne' Code_Saturne source directory,
but from a separate directory.

We recommend running for example:

  cd /home/<user>/code_saturne/..
  mkdir saturne_build
  cd saturne_build
  ../code_saturne/install_saturne.py

or (using absolute paths):

  mkdir /home/<user>/code_saturne_build
  cd /home/<user>/code_saturne_build
  /home/<user>/code_saturne/install_saturne.py
```
Way of relative paths addresses nonexisting file from saturne_build perspective. Way of absolute paths finishes like this:
```
user@host:~/code_saturne_build$ /home/user/code_saturne/install_saturne.py 

        Installation of Code_Saturne
        ____________________________

The process will take several minutes.
You can have a look at the log file meanwhile.
Traceback (most recent call last):
  File "/home/user/code_saturne/install_saturne.py", line 1344, in <module>
    setup = Setup()
  File "/home/user/code_saturne/install_saturne.py", line 586, in __init__
    p.set_version_from_configure(os.path.join(self.top_srcdir, 'configure'))
  File "/home/user/code_saturne/install_saturne.py", line 203, in set_version_from_configure
    f = open(path)
IOError: [Errno 2] No such file or directory: '/home/user/code_saturne/configure'
```
Another file, "configure.ac" is available in "code_saturne".

**Comment by YvanFournier** [comment]
We mostly need to specify in the README.md that when cloning the Git repository, `./sbin/bootstrap` needs to be run first, to generate `configure` and `Makefile.in` files (which are pre-gernerated when downloading a release `.tar.gz` from the main website).

This is specified in the detailed pdf documentation, bu since the README.md provides simple install instructions, it should also be there...

**Comment by xanfus** [comment]
> This is specified in the detailed pdf documentation, but since the README.md provides simple install instructions, it should also be there...
Thank you for your swift response. I went with PDF documentation for a while and made a break, when successive calls _configure_ with CC and FC parameters set (page 12, v6.0) finished with no HDF5 support (--with-med wasn't set either). Without further investigation, I realised, i can go with prebuilt apt package with code-saturne v4.3.3 starting from today, as the first acquaintance. v6.0.2 is prepared for Ubuntu 20.04, which i consider too.
Somehow, i missed a mention about code-saturne distribution in package manager at web site.
